.gitignore

```diff
@@ -301,3 +301,10 @@ TSWLatexianTemp*
 #*Notes.bib
 
 old
+
+# 忽略所有的 .DS_Store 文件
+.DS_Store
+
+# 忽略 papers 目录下的 _book 目录，不忽略主 cadp_docs 目录下的 _book
+/papers/_book/
```

Comment:
This is incorrect. Based on the path structure, the correct syntax to ignore `_book` directories within the `papers` directory is:

```
papers/**/_book
```

You can do the same thing to ignore the `libs` directory.

Since you have already added those files, you need to remove them from the commit by deleting them. When you compile the paper, they will be recreated.

Reply:
https://git-scm.com/docs/gitignore

Reply:
Yes Professor. I guess I forget to delete the syntax I wrote before and I will delete them. This is the syntax I'm using now:
.DS_Store
papers/**/_book/
papers/**/lib/
papers/**/_bookdown_files/
*.html
*.tex
*.log
I wonder if I need to add *.html, *.tex, *.log to .gitignore. 

Apart from this, I'm thinking if we need to show the paper as a html format under papers in github?